### PR TITLE
release v3.0.1

### LINKED ISSUE #787: release v3.0.1
once #786 is merged we should release a new point release to address bugs in capa

### PR DESCRIPTION
**Pull request by mr-tz**
closes #787 

# Release checklist

- [x] Ensure all [milestoned issues/PRs](https://github.com/fireeye/capa/milestones) are addressed, or reassign to a new milestone.
- [x] Add the `dont merge` label to all PRs that are close to be ready to merge (or merge them if they are ready) in [capa](https://github.com/fireeye/capa/pulls) and [capa-rules](https://github.com/fireeye/capa-rules/pulls).
- [x] Ensure the [CI workflow succeeds in master](https://github.com/fireeye/capa/actions/workflows/tests.yml?query=branch%3Amaster).
- [x] Ensure that `python scripts/lint.py rules/ --thorough` succeeds (only `missing examples`  offenses are allowed in the nursery).
- [x] Review changes
  - capa https://github.com/fireeye/capa/compare/\<last-release\>...master
  - capa-rules https://github.com/fireeye/capa-rules/compare/\<last-release>\...master
- [x] Update [CHANGELOG.md](https://github.com/fireeye/capa/blob/master/CHANGELOG.md)
  - Do not forget to add a nice introduction thanking contributors
  - Remember that we need a major release if we introduce breaking changes
  - Sections: see template below
  - Update `Raw diffs` links
  - Create placeholder for `master (unreleased)` section
    ```
    ## master (unreleased)

    ### New Features

    ### Breaking Changes

    ### New Rules (0)

    -

    ### Bug Fixes

    ### capa explorer IDA Pro plugin

    ### Development

    ### Raw diffs
    - [capa <release>...master](https://github.com/fireeye/capa/compare/<release>...master)
    - [capa-rules <release>...master](https://github.com/fireeye/capa-rules/compare/<release>...master)
    ```
- [x] Update [capa/version.py](https://github.com/fireeye/capa/blob/master/capa/version.py)
- [x] Create a PR with the updated [CHANGELOG.md](https://github.com/fireeye/capa/blob/master/CHANGELOG.md) and [capa/version.py](https://github.com/fireeye/capa/blob/master/capa/version.py). Copy this checklist in the PR description.
- [ ] After PR review, merge the PR and [create the release in GH](https://github.com/fireeye/capa/releases/new) using text from the [CHANGELOG.md](https://github.com/fireeye/capa/blob/master/CHANGELOG.md).
- [ ] Verify GH actions [upload artifacts](https://github.com/fireeye/capa/releases), [publish to PyPI](https://pypi.org/project/flare-capa) and [create a tag in capa rules](https://github.com/fireeye/capa-rules/tags) upon completion.
- [ ] [Spread the word](https://twitter.com)
- [ ] Update internal service

### Checklist

- [ ] No CHANGELOG update needed

- [x] No new tests needed

- [x] No documentation update needed

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,15 +12,27 @@
...
 
 ### Raw diffs
-- [capa <release>...master](https://github.com/fireeye/capa/compare/v3.0.0...master)
-- [capa-rules <release>...master](https://github.com/fireeye/capa-rules/compare/v3.0.0...master)
+- [capa v3.0.1...master](https://github.com/fireeye/capa/compare/v3.0.1...master)
+- [capa-rules v3.0.1...master](https://github.com/fireeye/capa-rules/compare/v3.0.1...master)
+
+## v3.0.1 (2021-09-27)
+
+This version updates the vivisect dependency. Users will experience less bugs and improved analysis results.
```
Comment (williballenthin) [inline CHANGELOG.md:25]:
```suggestion
This version updates the version of vivisect used by capa. Users will experience fewer bugs and find improved analysis results.
```

**Review by williballenthin — APPROVED** [review]
awesome!

**Review by mike-hunhoff — APPROVED** [review]
LGTM 🚀

**Comment by mr-tz** [comment]
Thanks, finalizing this now!

**Pull request merged**